python/python/glide/async_commands/cluster_commands.py

```diff
@@ -920,6 +920,64 @@ async def publish(
         )
         return cast(int, result)
 
+    async def pubsub_shardchannels(
+        self, pattern: Optional[TEncodable] = None
+    ) -> List[bytes]:
```

Comment:
why List[bytes] and not List[TEncodable] ?

Reply:
Python is returning only bytes for now. Similar to other commands

Reply:
why is the pattern given as TEncodable? why isnt it symmetric ?

Reply:
I think because we want node to be able to return bytes but to return string as default because bytes in node are not as nice as bytes in python